configure

```diff
@@ -265,7 +265,7 @@ if test "$gcc" -eq 1 && ($cc $CFLAGS -c $test.c) >> configure.log 2>&1; then
   CFLAGS="${CFLAGS} -std=c99"
 
   # Re-check ARCH if the compiler is a cross-compiler.
-  if $CC -print-multiarch && test -n "$($CC -print-multiarch)" 1> /dev/null 2>&1; then
+  if $CC -print-multiarch 1> /dev/null 2>&1 && test -n "$($CC -print-multiarch)" 1> /dev/null 2>&1; then
```

Comment:
The redirection to /dev/null does not hide the error, the return code of executing `$CC -print-multiarch` is used to find out whether the compiler `$CC` supports that flag.  clang for example does not support it.